build.gradle

```diff
@@ -985,72 +938,80 @@ task updater (dependsOn: [setupJar, widgetSetJar, allModules], type: Zip){
         include '*'
         include '**/*'
     }
-    from ("${project.buildDir}/libs") {
+    from("${project.buildDir}/libs") {
         include 'opencms.jar'
         include 'opencms-setup.jar'
         include 'opencms-widgetset.jar'
         into '/WEB-INF/lib'
     }
-    into ('/WEB-INF/lib') { from configurations.distribution }
-    tasks.findAll{dist_task -> dist_task.name.startsWith('dist_') && !(updater_excluded_modules.split(',').contains(dist_task.moduleName))}.each{ dist_task ->
-        from ("${modulesDistsDir}/${dist_task.archiveName}"){ into '/update/modules' }
-        if (dist_task.requiresJar.toBoolean()){
-            from ("${project.buildDir}/libs/${dist_task.moduleName}.jar"){ into 'WEB-INF/lib' }
+    into('/WEB-INF/lib') { from configurations.distribution }
+    tasks.findAll { dist_task -> dist_task.name.startsWith('dist_') && !(updater_excluded_modules.split(',').contains(dist_task.moduleName)) }.each { dist_task ->
+        from("${modulesDistsDir}/${dist_task.archiveName}") { into '/update/modules' }
+        if (dist_task.requiresJar.toBoolean()) {
+            from("${project.buildDir}/libs/${dist_task.moduleName}.jar") { into 'WEB-INF/lib' }
         }
     }
-    fileTree('webapp/setup/database'){ include '**/*.jar'}.each {File driverJar ->
+    fileTree('webapp/setup/database') { include '**/*.jar' }.each { File driverJar ->
         // copy the database drivers into the lib folder
-        from (driverJar.path) { into '/WEB-INF/lib' }
+        from(driverJar.path) { into '/WEB-INF/lib' }
+    }
+    Properties buildProps = new Properties()
+    buildProps.load(new FileInputStream("${projectDir}/build-default.properties"))
+    buildProps['updater.jars.remove'].split(',').each { jar ->
+        from("${projectDir}/webapp/update/empty.jar") {
+            into '/WEB-INF/lib'
+            rename('empty\\.jar', jar)
+        }
     }
-    
 }
 
-updater.dependsOn(tasks.matching{dist_task -> dist_task.name.startsWith('dist_') && !(updater_excluded_modules.split(',').contains(dist_task.moduleName))})
+updater.dependsOn(tasks.matching { dist_task -> dist_task.name.startsWith('dist_') && !(updater_excluded_modules.split(',').contains(dist_task.moduleName)) })
 
-task bindist (dependsOn: war, type: Zip){
+task bindist(dependsOn: war, type: Zip) {
+    group = "dist"
     baseName 'opencms'
     from "${project.buildDir}/distributions/opencms.war"
     from(projectDir) {
         include 'install.html'
         include 'license.txt'
         include 'history.txt'
         filter(ReplaceTokens, tokens: [
-            OPENCMS_VERSION_NUMBER: version,
-            OPENCMS_VERSION: product_name+' '+version
+                OPENCMS_VERSION_NUMBER: version,
+                OPENCMS_VERSION       : product_name + ' ' + version
         ])
     }
 }
 
 
 test {
-    classpath += fileTree('webapp/setup/database'){ include '**/*.jar' }
-    classpath += files("$buildDir/classes/setup") { builtBy 'setupClasses' }
```

Comment:
`setupClasses` has no outputs. Additionally:

 - `$buildir/classes/*` is output of `compile*Java` (not `*Classes`).
 - `$builddir/resources/*` is output of `process*Resources` (but it is not used)

, so the appropriate substitution is the output files of the `compile*Java` tasks.